src/dmd/backend/cgsched.d

```diff
@@ -1759,18 +1765,19 @@ else
             if (irm != modregrm(0,0,6))
             {
                 switch (mod)
-                {   case 0:
-                        c.IFL1 = FLconst;
-                        c.IEV1.Vpointer = 0;
-                        irm |= 0x80;
-                        break;
-                    case 1:
-                        c.IEV1.Vpointer = cast(byte) c.IEV1.Vpointer;
-                        irm = modregrm(2,0,rm);
-                        break;
+                {
+                case 0:
```

Comment:
I never liked this style of indentation. Please don't.

Reply:
Fixed